src/player_activity.cpp

```diff
@@ -24,6 +25,24 @@ player_activity::player_activity( activity_id t, int turns, int Index, int pos,
 {
 }
 
+void player_activity::migrate_item_position( Character &guy )
+{
+    const bool simple_action_replace =
+        type == ACT_FIRSTAID || type == ACT_GAME ||
+        type == ACT_PICKAXE || type == ACT_START_FIRE ||
+        type == ACT_HAND_CRANK || type == ACT_VIBE ||
+        type == ACT_OXYTORCH || type == ACT_FISH ||
+        type == ACT_ATM;
+
+    if( simple_action_replace ) {
+        targets.push_back( item_location( guy, &guy.i_at( position ) ) );
+    } else if( type == ACT_GUNMOD_ADD ) {
```

Comment:
imo a switch statement with fall through would be much cleaner here.

Reply:
can't use a switch for strings in C++ i thought?

Reply:
Ugh, yeah forgot that `type` is a string. C++ can be quite annoying at times.